e2e/geomap-suite/geomap-layer-types.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { e2e } from '@grafana/e2e';
+
+const DASHBOARD_ID = 'P2jR04WVk';
+
+e2e.scenario({
+  describeName: 'Geomap layer types',
+  itName: 'Tests changing the layer type',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.openDashboard({ uid: DASHBOARD_ID, queryParams: { editPanel: 1 } });
+    cy.get('[data-testid="layer-drag-drop-list"]').should('be.visible');
+    e2e.components.PanelEditor.OptionsPane.fieldLabel('Map layers Layer type').should('be.visible');
```

Comment:
How do we determine what the fieldLabel is? Is there an easy way to share this across both our test(s) and where we are referencing this in code? Ideally we should try and avoid breaking tests from changing text / messaging

If nothing else we probably could consolidate the `Map layers Layer type` into a const at top of file given it is referenced multiple times (as well as `Map layers Data` etc.)

Reply:
Updated with some constants. 
`fieldLabel` is based on the field `id` - [OptionsPanelItemDescriptor](https://github.com/grafana/grafana/blob/02b71ddfb5c5977a0be01af34fd00e8478cdec86/public/app/features/dashboard/components/PanelEditor/OptionsPaneItemDescriptor.tsx). It seems to be the way to go for every field/test related to the options panel.